examples/pump-controller-app/ti/cc13x4_26x4/args.gni

```diff
@@ -45,10 +45,16 @@ chip_openthread_target =
 
 chip_openthread_ftd = true
 
+#NOTE: LwIP is no longer supported, these options must stay the same
+chip_system_config_use_openthread_inet_endpoints = true
+chip_with_lwip = false
+chip_enable_openthread = true
+chip_inet_config_enable_tcp_endpoint = false
+
 # Disable CHIP Logging
-#chip_progress_logging = false
-#chip_detail_logging = false
-#chip_automation_logging = false
+#chip_progress_logging = true
```

Comment:
These are commented out - remove alltogether? Seems like a noop and comment does not match anymore even.

Reply:
This should have been enabled, just pushed the changes re-enabling logging